src/ThreeEditor/Simulation/Scoring/ScoringOutputTypes.ts

```diff
@@ -138,3 +138,457 @@ export const MEDIUM_KEYWORD_OPTIONS = Object.keys(MEDIUM_KEYWORDS).reduce(
 	},
 	{} as Record<MEDIUM, MEDIUM>
 );
+
+export const SCORING_OPTIONS = {
+	FLUKA: {
+		DETECTOR: {
+			'1MeVNEq': {
+				configuration: new Set(['MEDIUM', 'PER_PRIMARY']),
+				modifiers: new Set([
+					DETECTOR_MODIFIERS_DESCRIPTION.ANGLE,
+					DETECTOR_MODIFIERS_DESCRIPTION.DEDX,
+					DETECTOR_MODIFIERS_DESCRIPTION.E,
+					DETECTOR_MODIFIERS_DESCRIPTION.EAMU,
+					DETECTOR_MODIFIERS_DESCRIPTION.ENUC,
+					DETECTOR_MODIFIERS_DESCRIPTION.MDEDX,
+					DETECTOR_MODIFIERS_DESCRIPTION.TL,
+					DETECTOR_MODIFIERS_DESCRIPTION.Z,
+					DETECTOR_MODIFIERS_DESCRIPTION.Zeff,
+					DETECTOR_MODIFIERS_DESCRIPTION.Z2Beta2,
+					DETECTOR_MODIFIERS_DESCRIPTION.Zeff2Beta2
+				])
+			},
```

Comment:
Shouldn't you swap FLUKA and SHIELD-HIT12A ? Fluka by no way can score "Alanine" or "LET"